src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2693,6 +2693,17 @@
 helps['bicep'] = """
 type: group
 short-summary: Bicep CLI command group.
+long-summary: |
+  Bicep CLI command group. There are two configurations that can be set for the command group,
+  including bicep.check_version and bicep.use_binary_from_path.
+
+  bicep.check_version accepts Boolean values and defaults to True. If set to False, version checks for Bicep CLI installed via Azure CLI will be disabled.
+
+  The possible values of bicep.use_binary_from_path are if_found_in_ci and Booleans. The default value is if_found_in_ci.
+
+  With if_found_in_ci, GitHub Actions and Azure Pipeline users may expect some delay (usually a few days) in getting the latest version of Bicep CLI, since the az bicep commands will use the pre-installed Bicep CLI on the build agents, but the build agents has a different release cycle. The benefit is that the az bicep commands will not download the Bicep CLI on each pipeline run.
```

Comment:
> if_found_in_ci

Please do not expose the details of code logic in the help description, which will confuse users.
It is recommended to write these contents into the corresponding code comments.

Reply:
Removed the details from the help message.